app/components/BreadcrumbNav.tsx

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
- [ ] Add missing copyright and license header.
- [ ] Move to the route module folder, since this component is only used once.